metricflow/sql/render/big_query.py

```diff
@@ -129,6 +130,17 @@ def visit_date_trunc_expr(self, node: SqlDateTruncExpression) -> SqlExpressionRe
             bind_parameters=arg_rendered.bind_parameters,
         )
 
+    @override
+    def render_date_part(self, date_part: DatePart) -> str:
+        if date_part == DatePart.DOY:
+            return "dayofyear"
+        if date_part == DatePart.DOW:
+            return "dayofweek"
+        if date_part == DatePart.WEEK:
+            return "isoweek"
```

Comment:
The other engines return the value of the enum (so `dow`, `doy`, and `week`). BQ is the only one that doesn't support `dow` and `doy`, and the only one that we use `isoweek` for so it gets the special logic!

Reply:
Sorry, I meant if the other engines use the equivalent of isoweek vs some other week numbering standard.

And indeed they do not, because everything is terrible:

https://docs.snowflake.com/en/sql-reference/functions/year#usage-notes